### PR TITLE
toleration option added to all charts

### PR DESCRIPTION
**Pull request by DStorck**
Before I do this to ~30 charts, are we all okay with this format for adding tolerations ?

### CODE REVIEW

**Comment by coffeepac** [comment]
👍

**Comment by DStorck** [comment]
closes https://github.com/samsung-cnct/k2-charts/issues/112

File: zookeeper/templates/zookeeper.yaml [hunk 1]
```diff
@@ -109,6 +109,15 @@ spec:
...
         emptyDir: {}
+      {{- if .Values.tolerations }}
+      tolerations:
+      {{- range .Values.tolerations }}
+      - key: {{ .key | default ("") }}
+        value: {{ .value | default ("")}}
+        operator: {{ .operator | default ("Equal") }}
+        effect: {{ .effect  | default ("")}}
+      {{- end }}
+      {{- end }}
```
Comment (leahnp) [inline zookeeper/templates/zookeeper.yaml]: Is the indentation messed up here?

**Comment by leahnp** [comment]
LGTM

**Pull request merged**